script/version-utils.js

```diff
@@ -131,7 +143,7 @@ const versionBump = async (option, branchName) => {
   logger.log(`Do you want to upload these files to ${branchName} branch?\n`);
   await shouldContinuePrompt();
   spawnOrFail('git', [`push origin HEAD:${branchName} -f`]);
-  if (branchName === 'version-bump') {
+  if (branchName.startsWith('version-bump')) {
```

Comment:
Just for understanding, is there any case that the branch name will be `version-bump*`?

Reply:
No it should be version-bump-version (used to be just version-bump but that can cause conflicts when we try to release in multiple version.